### ISSUE TITLE
The example image 2

### ISSUE THREAD

**Issue by greatzh** [ntdat017/BusterNet_pytorch#6]
Sorry, this is not about the technical issue, I was just thinking the image in example 2 may not be suitable for everyone to view. Sometimes it may trigger some disgusting or other reaction.

**Comment by ntdat017** [comment]
yes, i think your concern is right, i will remove that example when I have free time.
